commands/remove.go

```diff
@@ -40,7 +40,12 @@ func runDelete(cmd *cobra.Command, args []string) error {
 	var services stack.Services
 	var gatewayAddress string
 	var yamlGateway string
-	if len(yamlFile) > 0 {
+
+	hasArgs := false
+
+	if len(args) == 1 {
```

Comment:
Finding the code that uses `hasArgs = true` hard to read.. can we make this simpler?